src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -77,23 +84,47 @@ function vertical_diffusion_boundary_layer_tendency!(
 )
     ᶜρ = Y.c.ρ
     FT = Spaces.undertype(axes(ᶜρ))
-    (; ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
-    (; C_E) = p.atmos.vert_diff
+    (; params) = p
+    thermo_params = CAP.thermodynamics_params(params)
+    (; ᶜts, ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
+    
+    z₀ = FT(3.21e-5)
+    f_b = FT(0.1)
+    κ = FT(0.4)
+    Ri_c = FT(1)
 
     ᶠgradᵥ = Operators.GradientC2F() # apply BCs to ᶜdivᵥ, which wraps ᶠgradᵥ
+    
+    ᶠρK_E = p.scratch.ᶠtemp_scalar
+    θ_v = p.scratch.ᶜtemp_scalar
```

Comment:
@simonbyrne I think my issue is related to the use of scratch variables here - theta_v is then used as an argument in line 115. Ri_a is a scalar field at the first interior center level.

Reply:
Assigning potential temperature columnwise to `@. θ_v[colidx]` now; and then calling `Fields.level(θ_v, 1)`to define `θ_a` seems to fix this issue. `deepcopy` is no longer used.